infra-as-code/bicep/modules/logging/mc-logging.parameters.example.json

```diff
@@ -35,6 +38,16 @@
       "value": {
           "Environment": "POC"
       }
+    },
+    "parAutomationAccountTags": {
+      "value": {
+          "Environment": "POC"
+      }
+    },
+    "parLogAnalyticsWorkspaceTags": {
```

Comment:
Let's emit/remove the `parAutomationAccountTags` and `parLogAnalyticsWorkspaceTags` and leave them to default